python/tskit/trees.py

```diff
@@ -2793,6 +2793,25 @@ def kc_distance(self, other, lambda_=0.0):
         """
         return self._ll_tree.get_kc_distance(other._ll_tree, lambda_)
 
+    def path_length(self, *args):
```

Comment:
The ``mrca`` function is a special case here because we can sensibly ask what's the MRCA of more than two nodes. However, I think we should fix this function to just supporting exactly two nodes, so let's change the signature to:
```python
def path_length(self, u, v):
```

Reply:
I was testing args length in the method to pass the tests `test_less_than_two_args` and `test_no_args` (copied from the `mrca` method). Do you think I keep those tests?

Reply:
No, can just drop them

Reply:
See suggestion below.